### PR TITLE
add os, build to travis, add Gitpod readme

### PR DESCRIPTION
**Pull request by mikedotexe**
Gitpod consistency, add os keys to Travis as well as test the build process.
Add README-Gitpod.yml that opens automatically.

### CODE REVIEW

**Comment by mikedotexe** [comment]
@willemneal same error here as:
https://github.com/near-examples/counter/pull/14#issuecomment-604737381

File: yarn.lock [hunk 1]
```diff
@@ -5031,7 +5031,7 @@ near-runtime-ts@^0.7.1-1:
   dependencies:
     near-sdk-as "^0.1.0-1"
 
-near-sdk-as@^0.1.0-1, near-sdk-as@^0.1.3:
+near-sdk-as@^0.1.0-1:
```
Comment (janedegtiareva) [inline yarn.lock]: hmm that looks like some temp preview version. Any idea where this came from?

Comment (mikedotexe) [reply yarn.lock]: weird, I'll rebuild

Comment (mikedotexe) [reply yarn.lock]: still don't know where this came from, but I'm not sure it's blocking anything

**Pull request merged**